python_functionality/homework.py

```diff
@@ -35,21 +43,33 @@ def task_3_find_item_via_value(data: DT, value) -> DT:
         find_item_via_value([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}], 26)
         >>> [{'name': 'Alex', 'age': 26}]
     """
-    pass
+    L = []
+    for i in range(len(data)):
+        if value in data[i].values():
+            L.append(data[i])
+    return L
 
 
 def task_4_min_value_integers(data: List[int]) -> int:
     """
     Find and return minimum value from list
     """
-    pass
+    if not data:
+        return None
+    else:
+        return min(data)
```

Comment:
you can simplify this part 
```suggestion
        if data:
```